src/modules/facility-loan/facility-loan.service.create-loan.test.ts

```diff
@@ -63,12 +72,16 @@ describe('FacilityLoanService', () => {
 
     describe('creates a bundle information in ACBS with a transformation of the requested new loan', () => {
       it('uses GBP dependent fields when request currency is GBP', async () => {
+        when(repaymentScheduleBuilderGetRepaymentSchedules).calledWith(newLoanGbp).mockReturnValueOnce(bondRepaymentSchedulesGbp);
+
         await service.createLoanForFacility(facilityIdentifier, newLoanGbp);
 
         expect(acbsBundleInformationServiceCreateBundleInformation).toHaveBeenCalledWith(acbsRequestBodyToCreateFacilityLoanGbp, idToken);
       });
 
       it('uses non-GBP dependent fields when request currency is not GBP', async () => {
+        when(repaymentScheduleBuilderGetRepaymentSchedules).calledWith(newLoanNonGbp).mockReturnValueOnce(bondRepaymentSchedulesGbp);
```

Comment:
Is it right that we're mocking to return `bondRepaymentSchedulesGbp` but testing `not GBP` here? I assume it doesn't actually matter because of how we're mocking, but it might be worth renaming `bondRepaymentSchedulesGbp` in this file to avoid confusion

Reply:
We could also have put this line in the `beforeEach`, since all the tests in the file use it

Reply:
Ah sorry, I missed that we couldn't use `beforeEach` - thanks!
Yep because of the way we're mocking I'm happy for us to just rename the variable :)